### ISSUE TITLE
Callback must be a function. Received undefined

### ISSUE THREAD

**Issue by pdehaan** [postlight/nodejs-typescript-kit#8]
On macOS Mojave (10.14.4) and getting the following errors w/ node v10.15.3 and v12.2.0:

```sh
$ nvm use --delete-prefix v10.15.3
Now using node v10.15.3 (npm v6.4.1)

$ npx @postlight/node-typescript-starter-kit my-awesome-project
Creating a new starter kit in /Users/pdehaan/dev/github/pdehaan/my-awesome-project

Callback must be a function
```

```sh
$ nvm i 12
Downloading and installing node v12.2.0...
Downloading https://nodejs.org/dist/v12.2.0/node-v12.2.0-darwin-x64.tar.xz...
######################################################################## 100.0%
Computing checksum with shasum -a 256
Checksums matched!
Now using node v12.2.0 (npm v6.9.0)

$ npx @postlight/node-typescript-starter-kit my-awesome-project
Creating a new starter kit in /Users/pdehaan/dev/github/pdehaan/my-awesome-project

Callback must be a function. Received undefined

$ echo $? # 0

**Comment by pdehaan** [comment]
Also possibly worth mentioning, but the blog post at https://postlight.com/trackchanges/introducing-postlights-node-js-typescript-starter-kit gives different install/usage information than the repo:

```sh
$ npx node-typescript-starter-kit my-awesome-project
npm ERR! code ENOVERSIONS
npm ERR! No valid versions available for node-typescript-starter-kit

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/pdehaan/.npm/_logs/2019-05-09T22_02_04_263Z-debug.log
Install for [ 'node-typescript-starter-kit@latest' ] failed with code 1
```

Note that the blog doesn't seem to include the `@postlight/` prefix for the npm org name:

```sh
$ npx node-typescript-starter-kit my-awesome-project # blog
$ npx @postlight/node-typescript-starter-kit my-awesome-project # readme
```

**Comment by bafxyz** [comment]
same here

**Comment by bafxyz** [comment]
here is PR https://github.com/postlight/nodejs-typescript-kit/pull/9

**Comment by aortbals** [comment]
Fixed in #9.
